flict/flictlib/project/reader.py

```diff
@@ -140,23 +146,30 @@ def read_project(self, project_file):
             "packages": flat_packages,
         }
 
+    def _relationship_is_dependency(self, relationshiptype):
+        return relationshiptype in DEPENDENCY_TAGS
+
     def _read_spdx_2_2(self, only_packages=None):
         project_name = self.project['name']
         packages = {}
         for pkg in self.project['packages']:
             elem_id = pkg['SPDXID']
             packages[elem_id] = {
                 'id': elem_id,
-                'name': pkg['name'],
-                'version': pkg['versionInfo'],
-                'license': pkg['licenseConcluded'],
-                'description': pkg['description'],
+                'name': pkg.get('name', ""),
```

Comment:
Same here - inconsistent quote style - please use single ticks only

Reply:
Additionally this strips the implied error handling - as the name from the package information can now be empty, which is wrong IMO - so the old style at least would have thrown an error

Reply:
reverted back away from get
quotes fixed

... in coming push